docs/modules/install/partials/version_matrix.adoc

```diff
@@ -0,0 +1,9 @@
+
+|===
+|Decidim version |Ruby version |Node version
+
+|v0.28 | 3.1.1 | 18.17.x
+
+|v0.27 | 3.0.2 | 16.18.x
+
```

Comment:
Just to be on the safe side ( a lot of users are still using decidim 0.26), and there are a lot of people involved in the development or bugfixes, i would add the following changes:

```suggestion
|Decidim version |Ruby version |Node version | Status

|develop | 3.2.2 | 18.17.x | Unreleased

|v0.28 | 3.1.1 | 18.17.x | Bug fixes and Security updates

|v0.27 | 3.0.2 | 16.18.x | Bug fixes and Security updates

|v0.26 | 2.7.5+ | 16.9.x | Not Maintained

```